feature-libs/smartedit/schematics/add-smartedit/index.ts

```diff
@@ -29,51 +23,11 @@ export function addSmartEditFeatures(options: SpartacusSmartEditOptions): Rule {
     const features = analyzeCrossFeatureDependencies(
       options.features as string[]
     );
-    const overrides = buildSmartEditConfig(options);
 
     return chain([
-      addFeatures(options, features, overrides),
+      analyzeInstalledFeatures(options, features),
+      addFeatures(options, features),
```

Comment:
👍  for normalization of adding features and making it possible with xxxes-schematics-config.ts